Specs/e2e/webgl-check.spec.js

```diff
@@ -0,0 +1,58 @@
+// tests/example1.test.js
+import { test } from "./test.js";
+
+function waitFor(delay) {
+  return new Promise((resolve) => setTimeout(resolve, delay));
+}
+
+const screenshotPath = "Specs/e2e/webgl-check";
+
+const chromeGpu = "chrome://gpu/";
+const webGlReport1 = "https://webglreport.com/?v=1";
+const webGlReport2 = "https://webglreport.com/?v=2";
+const aquarium = "https://webglsamples.org/aquarium/aquarium.html";
+
+test.beforeEach(async ({ page }, testInfo) => {
+  testInfo.setTimeout(testInfo.timeout + 160000);
+});
+
+/**
+ * This is used to check how WebGL is running in the testing environment to spot things like
```

Comment:
Is this a test we should commit, or one we should use for verification in this PR and then delete? I'm leaning towards the later since this PR should remove the need to have this test.

Reply:
I was a bit back and forth on it too but I thought it was more valuable to have in case we need to check in the future. I don't think it really does any harm being there. It definitely would've made my current investigation easier and now there'd be an easy thing to point to for anyone else having issues with the e2e tests?

I don't feel _super_ strongly about it and can take it out if you want

Reply:
After discussing with @jjhembd I decided to move this into the testing guide itself with a new section for e2e test performance instead of always keeping them around in an actual file.